.github/workflows/cleanup.yml

```diff
@@ -36,6 +36,14 @@ on:
         type: 'string'
         default: '1.21'
         required: false
+      pr_number:
```

Comment:
Need to make these two inputs as the end expire workflow needs to trigger this workflow, cannot make ${{ github.event.pull_request.number }} as default as it will fail if trigger is not pull_request event even with pr_number provided as input.